common/persistence/cassandraPersistence.go

```diff
@@ -2048,6 +2118,21 @@ func createRequestCancelInfo(result map[string]interface{}) *RequestCancelInfo {
 	return info
 }
 
+func createSerializedHistoryEventBatch(result map[string]interface{}) *SerializedHistoryEventBatch {
+	// TODO: default to JSON, update this when we support different encoding types.
+	eventBatch := &SerializedHistoryEventBatch{EncodingType: common.EncodingTypeJSON}
+	for k, v := range result {
+		switch k {
+		case "version":
```

Comment:
since the value is already set to JSON when create the struct in this method, so there is no need to read the encoding_type, unless we support other encoding type.